AEPMessaging/Sources/Messaging+EdgeEvents.swift

```diff
@@ -0,0 +1,220 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPCore
+import AEPServices
+import Foundation
+
+extension Messaging {
+    // MARK: - internal methods
+
+    /// Sends an experience event to the platform SDK for tracking the notification click-throughs
+    ///
+    /// - Parameters:
+    ///   - event: The triggering event with the click through data
+    ///   - config: configuration data
+    /// - Returns: A boolean explaining whether the handling of tracking info was successful or not
```

Comment:
at this moment this function does not return anything..
If we don't need it we can remove the return statement from the doc